src/styles/navstyles.css

```diff
@@ -16,11 +16,49 @@
   
   html,
   body {
-    font-family: "Segoe UI Variable Display", "Segoe UI", -apple-system, Roboto,
-      sans-serif, serif;
+    font-family: "Roboto Slab", serif;
     margin: 0;
+    font-weight: bold;
     /* overflow-x: hidden; */
   }
+
+  header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    background-color: #002a45;
+    z-index: 15;
+    
+  }
+
+
+  .nav-logo {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    margin-left: 0;
+    gap: 0.5rem;
+
+  }
+
+  h2 {
+    font-weight: 500;
+    font-size: xx-large;
+  }
+
+  nav {
+    display: flex;
+    align-items: center;
+  }
+
+  
+
+  h1, h2 {
```

Comment:
You already had styling for H2 on line 45 that gets overwritten by this code. Please remove the styling on H2 if you want to go with this styling.